apps/api/src/report/report.controller.ts

```diff
@@ -56,11 +60,17 @@ export class ReportController {
   @Get('/applicant/licensing-stage')
   @AllowAccess(Access.REPORTING)
   async getLicensingStageApplicants(
+    @Query('period') period: number,
     @Query('from') from: string,
     @Query('to') to: string,
+    @Query('testReport') testReport = false,
```

Comment:
This name is a little vague. It could be more specific like 'no-cache'

Reply:
Period and from/to seem conflicting with each other. What if `from` and `to` are not equal to the period's start and end?

Reply:
it's mainly for the no-cache call, I need the from and to for it, any ideas?

Reply:
AH, after having slept on it, I get what you mean... will fix